Makefile

```diff
@@ -197,7 +197,7 @@ MACOSX_SYSROOT  ?= $(shell xcrun --show-sdk-path)
 CC              := cc
 CXX             := c++
 CPP             := cc -E
-PATH            := /opt/procursus/bin:/opt/procursus/libexec/gnubin:/usr/bin:$(PATH)
+PATH            := /opt/procursus/bin:/opt/procursus/libexec/gnubin:$(PATH):/usr/bin
```

Comment:
I don't understand why this is needed. Also try not to patch the main Makefile in a subproject pull.

Reply:
Agreed.

Reply:
Sorry, I should have made a separate PR.
The reason for this is that when you don't have the bootstrap on a Mac but have tools installed from Brew, it keeps telling you to install GNU tar/dpkg/etc, even if you have them installed and in your PATH.
This is because /usr/bin is prioritized over your PATH, which it shouldn't be, because then it tries to use /usr/bin/make instead of /opt/homebrew/.../make, leading to an error when it could just use the tools installed from Brew to compile just fine.

Reply:
@Randomblock1 this is intentional. The as2 branch requires tools from Procursus. By the time it gets merged to master, there will be an easy install script + dependencies from Procursus Mac. This is for, in the future, it will be easier to do build dependencies per-project. If you'd care to join the discord, I can probably explain better.

Reply:
OK, I reverted the commit. What's the Discord invite?

Reply:
@Randomblock1 https://diatr.us/discord/